plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -294,13 +295,11 @@ def characters(s):
         character = encoded[start:end].decode('utf-32-be')
         yield character
 
-CGEventKeyboardSetUnicodeString = ctypes.cdll.LoadLibrary(ctypes.util.find_library('ApplicationServices')).CGEventKeyboardSetUnicodeString
-CGEventKeyboardSetUnicodeString.restype = None
 native_utf16 = 'utf-16-le' if sys.byteorder == 'little' else 'utf-16-be'
 
 def set_string(event, s):
     buf = s.encode(native_utf16)
-    CGEventKeyboardSetUnicodeString(objc.pyobjc_id(event), len(buf) / 2, buf)
+    CGEventKeyboardSetUnicodeString(event, len(buf) / 2, buf)
```

Comment:
A-hah! I thought that pyobjc_id thing looked suspicious, but didn't think to just try removing it. I was going to scour the PyObjC docs first. Failure to bias towards action on my part there!